Marlin/src/module/planner.cpp

```diff
@@ -2066,15 +2064,14 @@ bool Planner::_populate_block(block_t * const block, bool split_move,
   // Calculate and limit speed in mm/sec for each axis
   float current_speed[NUM_AXIS], speed_factor = 1.0f; // factor <1 decreases speed
   LOOP_XYZE(i) {
-    #if ENABLED(MIXING_EXTRUDER)
+    #if ENABLED(MIXING_EXTRUDER) && ENABLED(RETRACT_SYNC_MIXING)
```

Comment:
How is the max velocity of a e-stepper determined?
Usually you try to extrude faster and faster until the stepper skips. How fast it can go is determined by the moment of the stepper at a speed and it's 'resistance' id that speed. The 'resistance' is mainly the 'pressure' in the nozzle. Pressure is determined by the nozzles inner shape (diameter) and the viscosity of the material, what mainly depends on its temperature (nonlinear). Temperature depends mainly on the amount of material per time and the total power of the heater.
If you have more than one e-stepper feeding a nozzle they have to share the heater. It's impossible to extrude a MIXINGSTEPPERS times bigger amount of material.
Except when first retracting and then unretracting the same amount of material.

Limiting velocity to the amount what one spepper can push alone seems to be, at least, save.

Reply:
> I'm also concerned that delta_mm_i = delta_mm[i] / MIXING_STEPPERS will have unintended side effects since the planner now thinks that the speed of the extruder movement is now lower than it actually is.

Isn't that exactly what you wanted? The virtual tool for the autoretraction moves exactly the same amount of filament on all mixing steppers. So each of the mixing steppers has to move 1/MIXING_STEPPERS - and speed calculation is based on that.

Reply:
However.
Calculating a speed factor for every block is uneconomic. We could now do it one, when a new mix is stored in a virtual tool (normalizing) and stor it in a new float variable. That could be applied here.

Reply:
Could look like https://github.com/AnHardt/Marlin/commit/310788518f85eefb1746ed3d80c858338934699e
**UNTESTED !!!**

Reply:
>It's impossible to extrude a MIXINGSTEPPERS times bigger amount of material.
Except when first retracting and then unretracting the same amount of material.

Good point and great explanation. 

>Isn't that exactly what you wanted?

Yes. I just wasn't following the change correctly. 

>Could look like AnHardt@3280a60

Based on your explanation above, I don't think this is necessary.

Reply:
We could do it. But it would require an other method for finding the maximum possible speed. We'd have to run all mixing-steppers at once and at the same speed to find the maximum speed. Due to more power because of the multiple steppers we could get more pressure/flow - not MIXING_STEPPERS times more, but a bit. Then the sum of steps/time should be limited to that && no stepper should exceed it's individual max speed. (Not nice to calculate)